src/jmcomic/__init__.py

```
@@ -4,6 +4,8 @@
 
 __version__ = '2.7.5'
 
+from .jm_exception import DownloadCancelledException
+from .jm_runtime import *
```

Ruff (0.16.3)

[error] `from .jm_runtime import *` used; unable to detect undefined names

(F403)